src/ProjectTemplates/BlazorTemplates.Tests/PlaywrightFixture.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.BrowserTesting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging.Testing;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace ProjectTemplates.Tests.Infrastructure
+{
+    public class PlaywrightFixture<TTestAssemblyType> : IAsyncLifetime
```

Comment:
This seems general enough to go into some kind of shared location in the future if we're using the same infrastructure more broadly.

Reply:
Agree, but let's leave it here for now and move it when we have to